Makefile

```diff
@@ -16,59 +16,43 @@ install-mqa-transformers:
 
 # BLOOM AliBi
 hf-1b-bloom-fp32:
-	python src/main.py --hidden_size 2048 --n_head 16 --n_layer 24 --pipeline_class HF_GPU_Pipeline --model_class BLOOM --dtype float32 --batch_size ${batch_size}
+	python3 src/main.py --hidden_size 2048 --n_head 16 --n_layer 24 --pipeline_class HF_Pipeline --model_class BLOOM --dtype float32 --batch_size ${batch_size}
```

Comment:
@mayank31398 I updated the targets and removed the obsolete ones, but we should move them away from the makefile in favour of bundling them with the experiments.

Reply:
Agreed, will ditch it over time